orchestrator/gorc/src/commands/keys/cosmos/add.rs

```diff
@@ -0,0 +1,59 @@
+use super::show::ShowCosmosKeyCmd;
+use crate::application::APP;
+use abscissa_core::{Application, Command, Options, Runnable};
+use bip32;
+use k256::pkcs8::ToPrivateKey;
+use rand_core::OsRng;
+use signatory::FsKeyStore;
+use std::path;
+
+#[derive(Command, Debug, Default, Options)]
+pub struct AddCosmosKeyCmd {
+    #[options(free, help = "add [name]")]
+    pub args: Vec<String>,
+
+    #[options(help = "overwrite existing key")]
```

Comment:
```suggestion
    #[options(overwrite = "overwrite existing key")]
```

Reply:
This is an `--overwrite` flag right?

Reply:
Also this options correction is incorrect.

Reply:
Yup; that's providing help for a --overwrite flag. But I think our help system isn't generally working well.